source/includes/_executions.md

```diff
@@ -76,6 +76,9 @@ Property | Type | Description
 --------- | ------- | -----------
 `workflow_id` | string | The workflow id/name to execute.
 `deployment_id` | string | The id of the deployment the workflow should be executed on.
+`allow_custom_parameters` | boolean | Specifies whether to allow custom parameters, which are not present in the parameters schema of the workflow, to be passed when starting the execution (default=false).
+`parameters` | object | A dictionary containing parameters to be passed to the execution when starting it.
```

Comment:
in the `executions start` help we add some more details on the `parameters` option: 
`(formatted as YAML or as "key1=value1;key2=value2")`. Can we add that here as well?